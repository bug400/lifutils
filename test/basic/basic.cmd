@echo off
if exist liftest.dat del /F liftest.dat
if exist tst.rom del /F tst.rom

lifutils lifmod -v -e -f ..\data\VERMROM.MOD > test.txt
python ..\difftool.py  --binary VERM1.rom ..\data\VERM1.ROM
python ..\difftool.py test.txt ..\data\mod1_win.txt
copy ..\data\hp41cass.dat liffix.dat > nul
lifutils lifdir liffix.dat > test.txt
python ..\difftool.py test.txt ..\data\lifdir_before_fix.txt

lifutils liffix -m cass liffix.dat
lifutils lifdir liffix.dat > test.txt
python ..\difftool.py test.txt ..\data\lifdir_after_fix.txt

lifutils lifinit -m cass liftest.dat 60
lifutils liflabel liftest.dat TEST
lifutils lifdir liftest.dat > test.txt
python ..\difftool.py test.txt ..\data\lifdir_liftest_empty.txt

lifutils comp41 -x hpil -x hepax < ..\data\audi2.txt | lifutils raw41lif AUDI2 | lifutils lifput liftest.dat
lifutils comp41 -x hpil -f TEST1 -x hepax < ..\data\prog41.txt | lifutils lifput liftest.dat
lifutils textlif TXTA < ..\data\txta.txt | lifutils lifput liftest.dat
lifutils textlif -s 0 TXT41 < ..\data\txta.txt | lifutils lifput liftest.dat
lifutils textlif -s 42 TXT412 < ..\data\txta.txt | lifutils lifput liftest.dat
lifutils lifrename liftest.dat TXTA TXTB
lifutils textlif75 TXT75 < ..\data\txta.txt | lifutils lifput liftest.dat
lifutils textlif75 TXT75 < ..\data\txta.txt | lifutils lifput liftest.dat
lifutils textlif75 -n TXT75L < ..\data\txta75.txt | lifutils lifput liftest.dat
type VERM1.rom | lifutils rom41hx VERMROM | lifutils lifput liftest.dat
type VERM1.rom | lifutils rom41lif VERMROML | lifutils lifput liftest.dat
type VERM1.rom | lifutils rom41er VERMROME | lifutils lifput liftest.dat
type ..\data\txta.txt | lifutils textlif TXTA | lifutils lifput liftest.dat
lifutils lifput liftest.dat ..\data\dat1.lif
lifutils lifput liftest.dat ..\data\key1.lif
lifutils lifput liftest.dat ..\data\wall1.lif
lifutils lifput liftest.dat ..\data\stat1.lif
lifutils lifput liftest.dat ..\data\memt.lif
lifutils lifput liftest.dat ..\data\wall1_long.lif 2> err_long.txt
python ..\difftool.py err_long.txt ..\data\err_long.txt
lifutils lifput liftest.dat ..\data\wall1_short.lif 2> err_short.txt
python ..\difftool.py err_short.txt ..\data\err_short.txt
lifutils lifput liftest.dat ..\data\phycons.lif
lifutils lifstat liftest.dat > test.txt
python ..\difftool.py test.txt ..\data\lifstat_before_pack.txt

lifutils liflabel liftest.dat
lifutils lifdir liftest.dat > test.txt
python ..\difftool.py test.txt ..\data\lifdir_liftest_filled.txt

lifutils lifpurge liftest.dat VERMROM
lifutils lifpack liftest.dat
lifutils liflabel -c liftest.dat
lifutils lifstat liftest.dat > test.txt
python ..\difftool.py test.txt ..\data\lifstat_after_pack.txt

lifutils lifstat liftest.dat 97 > test.txt
python ..\difftool.py test.txt ..\data\lifstat_memt.txt
lifutils lifstat liftest.dat 3 0 2 > test.txt
python ..\difftool.py test.txt ..\data\lifstat_memt.txt

lifutils lifget -b liftest.dat MEM > temp.lif
lifutils lifpurge liftest.dat MEM
lifutils lifput liftest.dat temp.lif

lifutils lifdir liftest.dat > test.txt
python ..\difftool.py test.txt ..\data\lifdir_liftest_packed.txt
lifutils lifdir -n liftest.dat > test.txt
python ..\difftool.py test.txt ..\data\lifdir_liftest_packed_names.txt
lifutils lifdir -v 2 liftest.dat > test.txt
python ..\difftool.py test.txt ..\data\lifdir_liftest_packed_verbose.txt
lifutils lifdir -c liftest.dat > test.txt
python ..\difftool.py test.txt ..\data\lifdir_liftest_packed_csv.txt

lifutils lifget  liftest.dat TXTA  | lifutils lifraw | lifutils liftext > test.txt
python ..\difftool.py  test.txt ..\data\txta.txt

lifutils lifget  liftest.dat TXT75  | lifutils lifraw | lifutils liftext75  > test.txt
python ..\difftool.py  test.txt ..\data\txta.txt

lifutils lifget liftest.dat TXT75L | lifutils lifraw | lifutils liftext75 > test.txt
python ..\difftool.py  test.txt ..\data\txta.txt
lifutils lifget liftest.dat TXT75L | lifutils lifraw | lifutils liftext75 -n > test.txt
python ..\difftool.py  test.txt ..\data\txta75.txt


lifutils lifget -r liftest.dat TEST1 |  lifutils decomp41 -x hpil -x hepax > test.txt
python ..\difftool.py  test.txt ..\data\prog41.txt

lifutils lifget -r liftest.dat DAT1 | lifutils sdata > test.txt
python ..\difftool.py  test.txt ..\data\dat1.txt
lifutils lifget -r liftest.dat DAT1 | lifutils sdata -h > test.txt
python ..\difftool.py  test.txt ..\data\dat1_hex.txt
lifutils lifget -r liftest.dat DAT1 | lifutils sdata -n -b -l > test.txt
python ..\difftool.py  test.txt ..\data\dat1_extended.txt

lifutils lifget -r liftest.dat DAT1 | lifutils regs41 > test.txt
python ..\difftool.py  test.txt ..\data\regs1.txt

lifutils lifget -r liftest.dat KEY1 | lifutils key41 -x hpdevices > test.txt
python ..\difftool.py  test.txt ..\data\key1.txt
lifutils lifget -r liftest.dat KEY1 | lifutils key41 -h > test.txt
python ..\difftool.py  test.txt ..\data\key1_hex.txt

lifutils lifget -r liftest.dat STAT1 | lifutils stat41 > test.txt
python ..\difftool.py  test.txt ..\data\stat1.txt
lifutils lifget -r liftest.dat STAT1 | lifutils stat41 -b -f -v > test.txt
python ..\difftool.py  test.txt ..\data\stat1_extended.txt


lifutils lifget -r liftest.dat WALL1 | lifutils wcat41 > test.txt
python ..\difftool.py  test.txt ..\data\wcat1.txt

type VERM1.rom | lifutils rom41cat -x > test.txt
python ..\difftool.py  test.txt ..\data\rom41cat1.txt

lifutils lifget -r liftest.dat DAT1 | lifutils sdatabar | lifutils barps > test.ps
python ..\difftool.py  test.ps ..\data\sdata.ps

lifutils lifget -r liftest.dat TEST1 | lifutils prog41bar | lifutils barps > test.ps
python ..\difftool.py  test.ps ..\data\prog.ps

lifutils lifget -r liftest.dat DAT1 | lifutils sdatabar | lifutils barprt > test.txt
python ..\difftool.py  test.txt ..\data\barprt.txt

lifutils lifget -r liftest.dat MEM | lifutils lexcat71 > test.txt
python ..\difftool.py  test.txt ..\data\lexcat71.txt

lifutils lifget -r liftest.dat PHYCONS | lifutils lexcat75 > test.txt
python ..\difftool.py  test.txt ..\data\lexcat75.txt

lifutils lifheader ..\data\memt.lif > test.txt
python ..\difftool.py  test.txt ..\data\lifheader.txt

lifutils rom41hx VERMROM < VERM1.rom | lifutils lifraw | lifutils hx41rom > tst.rom
python ..\difftool.py  --binary VERM1.rom tst.rom

lifutils rom41er VERMROM < VERM1.rom | lifutils lifraw | lifutils er41rom > tst.rom
python ..\difftool.py  --binary VERM1.rom tst.rom

lifutils lifget -r liftest.dat AUDI2 | lifutils outp41 | lifutils inp41 | lifutils decomp41 -x hpil -x hepax > test.txt
python ..\difftool.py  test.txt ..\data\audi2.txt 

lifutils wall41 -r -k < ..\data\wall1.lif | key41 -x hpdevices > test.txt
python ..\difftool.py  test.txt ..\data\key1.txt
lifutils wall41 -r -g < ..\data\wall1.lif | sdata > test.txt
python ..\difftool.py  test.txt ..\data\dat1.txt
lifutils wall41 -r -g < ..\data\wall1.lif | regs41 > test.txt
python ..\difftool.py  test.txt ..\data\regs1.txt
lifutils wall41 -r -s < ..\data\wall1.lif | stat41 > test.txt
python ..\difftool.py  test.txt ..\data\stat1w.txt
lifutils wall41 -r -p prog < ..\data\wall1.lif 
lifutils decomp41 -x hpil -x hepax  < prog.001 > test.txt
python ..\difftool.py  test.txt ..\data\audi2.txt

if exist err_long.txt del /F err_long.txt
if exist err_short.txt del /F err_short.txt
if exist test.ps del /F test.ps
if exist test.txt del /F test.txt
if exist liftest.dat del /F liftest.dat
if exist tst.rom del /F tst.rom
if exist tst.ps del /F tst.ps
if exist temp.lif del /F temp.lif
if exist liffix.dat del /F liffix.dat
if exist prog.ps del /F prog.ps
if exist VERM1.rom del /f VERM1.rom
if exist tst.txt del /F tst.txt
if exist prog.000 del /F prog.000
if exist prog.001 del /F prog.001
if exist prog.002 del /F prog.002
