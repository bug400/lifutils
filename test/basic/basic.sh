rm -rf liftest.dat
rm -f tst.rom
lifutils lifmod -v -e -f ../data/VERMROM.MOD > test.txt
python3 ../difftool.py  --binary VERM1.rom ../data/VERM1.ROM
python3 ../difftool.py test.txt ../data/mod1_lin.txt
cp ../data/hp41cass.dat liffix.dat
lifutils lifdir liffix.dat > test.txt
python3 ../difftool.py test.txt ../data/lifdir_before_fix.txt

lifutils liffix -m cass liffix.dat
lifutils lifdir liffix.dat > test.txt
python3 ../difftool.py test.txt ../data/lifdir_after_fix.txt

lifutils lifinit -m cass liftest.dat 60
lifutils liflabel liftest.dat TEST
lifutils lifdir liftest.dat > test.txt
python3 ../difftool.py test.txt ../data/lifdir_liftest_empty.txt

lifutils comp41 -x hpil -x hepax < ../data/audi2.txt | lifutils raw41lif AUDI2 | lifutils lifput liftest.dat
lifutils comp41 -x hpil -x hepax -f TEST1 < ../data/prog41.txt | lifutils lifput liftest.dat
lifutils textlif TXTA < ../data/txta.txt | lifutils lifput liftest.dat
lifutils textlif -s 0 TXT41 < ../data/txta.txt | lifutils lifput liftest.dat
lifutils textlif -s 42 TXT412 < ../data/txta.txt | lifutils lifput liftest.dat
lifutils lifrename liftest.dat TXTA TXTB
lifutils textlif75 TXT75 < ../data/txta.txt | lifutils lifput liftest.dat
lifutils textlif75 -n TXT75L < ../data/txta75.txt | lifutils lifput liftest.dat
cat VERM1.rom | lifutils rom41hx VERMROM | lifutils lifput liftest.dat
cat VERM1.rom | lifutils rom41lif VERMROML | lifutils lifput liftest.dat
cat VERM1.rom | lifutils rom41er VERMROME | lifutils lifput liftest.dat
cat ../data/txta.txt | lifutils textlif TXTA | lifutils lifput liftest.dat
lifutils lifput liftest.dat ../data/dat1.lif
lifutils lifput liftest.dat ../data/key1.lif
lifutils lifput liftest.dat ../data/wall1.lif
lifutils lifput liftest.dat ../data/stat1.lif
lifutils lifput liftest.dat ../data/memt.lif
lifutils lifput liftest.dat ../data/wall1_long.lif 2> err_long.txt
python3 ../difftool.py err_long.txt ../data/err_long.txt
lifutils lifput liftest.dat ../data/wall1_short.lif 2> err_short.txt
python3 ../difftool.py err_short.txt ../data/err_short.txt
lifutils lifput liftest.dat ../data/phycons.lif
lifutils lifstat liftest.dat > test.txt
python3 ../difftool.py test.txt ../data/lifstat_before_pack.txt

lifutils liflabel liftest.dat
lifutils lifdir liftest.dat > test.txt
python3 ../difftool.py test.txt ../data/lifdir_liftest_filled.txt

lifutils lifpurge liftest.dat VERMROM
lifutils lifpack liftest.dat
lifutils liflabel -c liftest.dat
lifutils lifstat liftest.dat > test.txt
python3 ../difftool.py test.txt ../data/lifstat_after_pack.txt

lifutils lifstat liftest.dat 97 > test.txt
python3 ../difftool.py test.txt ../data/lifstat_memt.txt
lifutils lifstat liftest.dat 3 0 2 > test.txt
python3 ../difftool.py test.txt ../data/lifstat_memt.txt

lifutils lifget -b liftest.dat MEM > temp.lif
lifutils lifpurge liftest.dat MEM
lifutils lifput liftest.dat temp.lif

lifutils lifdir liftest.dat > test.txt
python3 ../difftool.py test.txt ../data/lifdir_liftest_packed.txt
lifutils lifdir -n liftest.dat > test.txt
python3 ../difftool.py test.txt ../data/lifdir_liftest_packed_names.txt
lifutils lifdir -v 2 liftest.dat > test.txt
python3 ../difftool.py test.txt ../data/lifdir_liftest_packed_verbose.txt
lifutils lifdir -c liftest.dat > test.txt
python3 ../difftool.py test.txt ../data/lifdir_liftest_packed_csv.txt

lifutils lifget liftest.dat TXTA | lifutils lifraw | lifutils liftext > test.txt
python3 ../difftool.py  test.txt ../data/txta.txt

lifutils lifget liftest.dat TXT75  | lifutils lifraw | lifutils liftext75  > test.txt
python3 ../difftool.py  test.txt ../data/txta.txt

lifutils lifget liftest.dat TXT75L | lifutils lifraw | lifutils liftext75 > test.txt
python3 ../difftool.py  test.txt ../data/txta.txt
lifutils lifget liftest.dat TXT75L | lifutils lifraw | lifutils liftext75 -n > test.txt
python3 ../difftool.py  test.txt ../data/txta75.txt

lifutils lifget -r liftest.dat TEST1 |  lifutils decomp41 -x hpil -x hepax > test.txt
python3 ../difftool.py  test.txt ../data/prog41.txt

lifutils lifget -r liftest.dat DAT1 | lifutils sdata > test.txt
python3 ../difftool.py  test.txt ../data/dat1.txt
lifutils lifget -r liftest.dat DAT1 | lifutils sdata -h > test.txt
python3 ../difftool.py  test.txt ../data/dat1_hex.txt
lifutils lifget -r liftest.dat DAT1 | lifutils sdata -n -b -l > test.txt
python3 ../difftool.py  test.txt ../data/dat1_extended.txt

lifutils lifget -r liftest.dat DAT1 | lifutils regs41 > test.txt
python3 ../difftool.py  test.txt ../data/regs1.txt

lifutils lifget -r liftest.dat KEY1 | lifutils key41 -x hpdevices > test.txt
python3 ../difftool.py  test.txt ../data/key1.txt
lifutils lifget -r liftest.dat KEY1 | lifutils key41 -h > test.txt
python3 ../difftool.py  test.txt ../data/key1_hex.txt

lifutils lifget -r liftest.dat STAT1 | lifutils stat41 > test.txt
python3 ../difftool.py  test.txt ../data/stat1.txt
lifutils lifget -r liftest.dat STAT1 | lifutils stat41 -b -f -v > test.txt
python3 ../difftool.py  test.txt ../data/stat1_extended.txt

lifutils lifget -r liftest.dat WALL1 | lifutils wcat41 > test.txt
python3 ../difftool.py  test.txt ../data/wcat1.txt

cat VERM1.rom | lifutils rom41cat -x > test.txt
python3 ../difftool.py  test.txt ../data/rom41cat1.txt

lifutils lifget -r liftest.dat DAT1 | lifutils sdatabar | lifutils barps > test.ps
python3 ../difftool.py --binary test.ps ../data/sdata.ps

lifutils lifget -r liftest.dat TEST1 | lifutils prog41bar | lifutils barps > test.ps
python3 ../difftool.py  test.ps ../data/prog.ps

lifutils lifget -r liftest.dat DAT1 | lifutils sdatabar | lifutils barprt > test.txt
python3 ../difftool.py  test.txt ../data/barprt.txt

lifutils lifget -r liftest.dat MEM | lifutils lexcat71 > test.txt
python3 ../difftool.py  test.txt ../data/lexcat71.txt

lifutils lifget -r liftest.dat PHYCONS | lifutils lexcat75 > test.txt
python3 ../difftool.py  test.txt ../data/lexcat75.txt

lifutils lifheader ../data/memt.lif > test.txt
python3 ../difftool.py  test.txt ../data/lifheader.txt

lifutils rom41hx VERMROM < VERM1.rom | lifutils lifraw | lifutils hx41rom > tst.rom
python3 ../difftool.py  --binary VERM1.rom tst.rom

lifutils rom41er VERMROM < VERM1.rom | lifutils lifraw | lifutils er41rom > tst.rom
python3 ../difftool.py  --binary VERM1.rom tst.rom

lifutils lifget -r liftest.dat AUDI2 | lifutils outp41 > test.txt
python3 ../difftool.py test.txt ../data/audi2.hex

lifutils lifget -r liftest.dat AUDI2 | lifutils outp41 | lifutils inp41 | lifutils decomp41 -x hpil -x hepax > test.txt
python3 ../difftool.py  test.txt ../data/audi2.txt 

lifutils wall41 -r -k < ../data/wall1.lif | key41 -x hpdevices > test.txt
python3 ../difftool.py  test.txt ../data/key1.txt
lifutils wall41 -r -g < ../data/wall1.lif | sdata > test.txt
python3 ../difftool.py  test.txt ../data/dat1.txt
lifutils wall41 -r -g < ../data/wall1.lif | regs41 > test.txt
python3 ../difftool.py  test.txt ../data/regs1.txt
lifutils wall41 -r -s < ../data/wall1.lif | stat41 > test.txt
python3 ../difftool.py  test.txt ../data/stat1w.txt
lifutils wall41 -r -p prog < ../data/wall1.lif 
lifutils decomp41 -x hpil -x hepax  < prog.001 > test.txt
python3 ../difftool.py  test.txt ../data/audi2.txt
rm -f err_long.txt
rm -f err_short.txt
rm -f liftest.dat
rm -f tst.rom
rm -f test.ps
rm -f temp.lif
rm -f VERM1.rom
rm -f liffix.dat
rm -f test.txt
rm -f prog.000
rm -f prog.001
rm -f prog.002
