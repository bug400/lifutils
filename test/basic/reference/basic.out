New Volume : TEST  
Volume : TEST   ( formatted : 01/01/01 00:00:00 )
Tracks: 2 Surfaces: 1 Blocks per Track: 256 Total size: 512 Blocks, 131072 Bytes
Directory start : 2 (0/0/3) end : 9 (0/0/10)
User data start : 10 (0/0/11) end : 55 (1/1/8)
Old Volume : TEST  
Volume : TEST   , formatted : 01/01/01 00:00:00
Tracks: 2 Surfaces: 1 Blocks/Track: 256 Total size: 512 Blocks, 131072 Bytes
AUDI2       PGM41        1972/2048                      
TEST1       PGM41         185/256                       
TXTB        TEXT          256/256      01/01/01 00:00:00
TXT75       TXT75         256/256      01/01/01 00:00:00
VERMROM     SDATA        5120/5120     01/01/01 00:00:00
TXTA        TEXT          256/256      01/01/01 00:00:00
DATA1       SDATA         160/256                       
KEY1        KEY41           9/256                       
WALL1       ALL41        2689/2816                      
STAT1       STAT41         81/256                       
10 files (64 max), last block used: 55 of 512
Volume : TEST   ( formatted : 01/01/01 00:00:00 )
Tracks: 2 Surfaces: 1 Blocks per Track: 256 Total size: 512 Blocks, 131072 Bytes
Directory start : 2 (0/0/3) end : 9 (0/0/10)
User data start : 10 (0/0/11) end : 35 (1/0/4)
Volume : TEST   , formatted : 01/01/01 00:00:00
Tracks: 2 Surfaces: 1 Blocks/Track: 256 Total size: 512 Blocks, 131072 Bytes
AUDI2       PGM41        1972/2048                      
TEST1       PGM41         185/256                       
TXTB        TEXT          256/256      01/01/01 00:00:00
TXT75       TXT75         256/256      01/01/01 00:00:00
TXTA        TEXT          256/256      01/01/01 00:00:00
DATA1       SDATA         160/256                       
KEY1        KEY41           9/256                       
WALL1       ALL41        2689/2816                      
STAT1       STAT41         81/256                       
9 files (64 max), last block used: 35 of 512
output of liftext...
RECORD1
RECORD2
RECORD3
output of liftext75...
0000 RECORD1
0001 RECORD2
0002 RECORD3
output of decomp41...
LBL "TEST1"
20
"TXTA"
HCRFLAS
"RECORD1"
HAPPREC
"RECORD2"
HAPPREC
"RECORD3"
HAPPREC
"TXTA,TXTA"
HWRTFL
"TXTA"
HPURFL
"TXTB,TXTB"
HREADFL
"TXTB"
11
STO 10
LBL 00
1
ST- 10
RCL 10
X=0?
GTO 01
HGETREC
AVIEW
GTO 00
LBL 01
"TXTB"
HPURFL
10
STO 00
LBL 02
RCL 00
STO IND 10
ST+ 10
RCL 10
20
X=Y?
GTO 03
GTO 02
LBL 03
"DATA1"
20
CREATE
0
SEEKR
10.020
WRTRX
"READP"
21
XF
8
"KEY1"
WRTK
"STAT1"
WRTS
"WALL1"
WRTA
END
Output of sdata...
+2.000000000E+1
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
+0.000000000E+0
Output of regs41...
0000 : 02 00 00 00 00 00 01
0001 : 00 00 00 00 00 00 00
0002 : 00 00 00 00 00 00 00
0003 : 00 00 00 00 00 00 00
0004 : 00 00 00 00 00 00 00
0005 : 00 00 00 00 00 00 00
0006 : 00 00 00 00 00 00 00
0007 : 00 00 00 00 00 00 00
0008 : 00 00 00 00 00 00 00
0009 : 00 00 00 00 00 00 00
000a : 00 00 00 00 00 00 00
000b : 00 00 00 00 00 00 00
000c : 00 00 00 00 00 00 00
000d : 00 00 00 00 00 00 00
000e : 00 00 00 00 00 00 00
000f : 00 00 00 00 00 00 00
0010 : 00 00 00 00 00 00 00
0011 : 00 00 00 00 00 00 00
0012 : 00 00 00 00 00 00 00
0013 : 00 00 00 00 00 00 00
Output of key41...
 21 : XROM 28,07
Output of wcat41...
SIZE 273
First statistical register = 011

1 KAR found
"TEST1"
.END.
Output of stat41...
RPN Stack : 

 T : +2.000000000E+1
 Z : +0.000000000E+0
 Y : +1.002000000E+1
 X : +2.100000000E+1
 L : +0.000000000E+0

Alpha : "STAT1"

SIZE = 273
Statistics start = 011

User Flags 00--43 : 00000000000000000000010100101100000001001000
Output of ROM dump
30 0 VERMROM_1A
30 1 XROM'AUDI2'
30 2 XROM'Z'
30 3 XROM'NCS'
30 4 XROM'PCS'
30 5 XROM'NEW'
30 6 XROM'PUR'
30 7 XROM'SAV'
30 8 XROM'GET'
30 9 XROM'?'
30 10 XROM'EIN'
30 11 XROM'AUS'
30 12 XROM'CLR'
30 13 XROM'INH'
30 14 XROM'AK'
30 15 XROM'WR'
30 16 XROM'CAN'
30 17 XROM'AN'
30 18 XROM'NAN'
30 19 XROM'CLA'
30 20 XROM'VPT'
30 21 XROM'DPT'
30 22 XROM'AUF'
30 23 XROM'AB'
30 24 XROM'ABL'
30 25 XROM'ABO'
30 26 XROM'TAV'
30 27 XROM'KLP'
30 28 XROM'FL'
30 29 XROM'FLP'
30 30 XROM'GS'
Row :  1 ,  8 bytes
HEX :  3b  62  b0  00  00  00  0e  1a
DEC :  59  98 176   0   0   0  14  26
Row :  2 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row :  3 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row :  4 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row :  5 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row :  6 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row :  7 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row :  8 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row :  9 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 10 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 11 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 12 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 13 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 14 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 15 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 16 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 17 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 18 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 19 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
Row : 20 ,  7 bytes
HEX :  1b  60  b0  00  00  00  0a
DEC :  27  96 176   0   0   0  10
output of lex71
LEX ID: 5c
Lowest token: 1
Highest token: 9
Poll handler present
 Keyword Token Characterization
    ANZ?     1 BASIC function
     DEL     2 Legal after THEN/ELSE, Programmable
     GET     3 Legal after THEN/ELSE, Programmable
    MAX?     4 BASIC function
    NEWK     5 Legal after THEN/ELSE, Programmable
     OUT     6 Legal after THEN/ELSE, Programmable
    PURK     7 Legal after THEN/ELSE, Programmable
     PUT     8 Legal after THEN/ELSE, Programmable
     UPD     9 Legal after THEN/ELSE, Programmable

output of lex75
LEX ID: 232 (hex)

Keywords:
FNC$: UNIT$
FNC#: VLITE
FNC#: CHARGE
FNC#: PLANCK
FNC#: ELMASS
FNC#: AVGDRO
FNC#: MOLRGC
FNC#: BOLTK
FNC#: GRAVK
FNC#: MOLVOL
FNC#: FARADK
FNC#: RYDBRK
FNC#: FINSTK
FNC#: ELRAD
FNC#: BOHRAD
FNC#: MAGFLX
FNC#: ATMASS
FNC#: PRMASS
FNC#: NTMASS
FNC#: ELGK
FNC#: BOHRMG
FNC#: NUCLMG
FNC#: ELMAG
FNC#: PRMAG
FNC#: PRGYMR


Code attributes:
RAMable
Position independent code
Mergeable
Lex identifier number independent
File name           : MEM       
File type           : e208 (LEX71) 
Data length         : 959
Number of blocks    : 4
Creation date       : 01/03/16 21:15:40
Implementation bytes: 80017D070000
